lib/model/model.go

```diff
@@ -1242,65 +1241,55 @@ func (m *Model) ConnectedTo(deviceID protocol.DeviceID) bool {
 }
 
 func (m *Model) GetIgnores(folder string) ([]string, []string, error) {
-	var lines []string
-
 	m.fmut.RLock()
 	cfg, ok := m.folderCfgs[folder]
 	m.fmut.RUnlock()
 	if !ok {
-		return lines, nil, fmt.Errorf("Folder %s does not exist", folder)
-	}
-
-	if !cfg.HasMarker() {
-		return lines, nil, fmt.Errorf("Folder %s stopped", folder)
-	}
-
-	fd, err := os.Open(filepath.Join(cfg.Path(), ".stignore"))
-	if err != nil {
-		if os.IsNotExist(err) {
-			return lines, nil, nil
+		cfg, ok = m.cfg.Folders()[folder]
+		if ok {
```

Comment:
I guess I'd invert the if condition doing the following:

```
runner, ok  = foo
if ok {
   return runner.Lines()
}

cfg, ok := getfolder
if ok {
  return readFile()
}
return nil
```

This way you don't have to follow to levels of ok's and terminates early which makes it more readable.